.env.example

```diff
@@ -0,0 +1,6 @@
+ PGUSER=postgres
```

Comment:
You dont need this as well as the `.env.example` file, one is enough to show what variable names are needed